src/optimize/babel/helpers.js

```diff
@@ -0,0 +1,54 @@
+// this file is not transpiled in dev
+
+const env = process.env;
+const fromRoot = require('path').resolve.bind(null, __dirname, '../../../');
+
+if (!env.BABEL_CACHE_PATH) {
+  env.BABEL_CACHE_PATH = fromRoot('optimize/.babelcache.json');
+}
+
+if (!env.WEBPACK_BABEL_CACHE_DIR) {
+  env.WEBPACK_BABEL_CACHE_DIR = fromRoot('optimize/.webpack.babelcache');
```

Comment:
Good catch, I enabled it in dev, but it's disabled in the distributable because it creates a directory and we won't be able to ensure all of it's files are `chown`ed